src/Service/UserBookingCacheService.php

```diff
@@ -86,10 +86,10 @@ public function updateCache($removeOutdated = true): void
     /**
      * {@inheritdoc}
      */
-    public function addCacheEntry(UserBooking $userBooking): void
+    public function addCacheEntry(UserBooking|array $booking): void
```

Comment:
I am not a big fan of this. Can you make a UserBooking out of the array, so you avoid handling different input.
Otherwise, make it into two different functions.

Reply:
Changed